actions/class.Items.php

```diff
@@ -38,70 +39,55 @@
  */
 class taoItems_actions_Items extends tao_actions_SaSModule
 {
-
+    use OntologyAwareTrait;
     use EventManagerAwareTrait;
 
-    /**
-     * constructor: initialize the service and the default data
-     * @return  Items
-     */
-    public function __construct(){
-
-        parent::__construct();
-
-        //the service is initialized by default
-        $this->service = taoItems_models_classes_ItemsService::singleton();
```

Comment:
In child class we call `$this->service` directly two times. It may fail if we do not instantiate it in constructor or replace to `$this->getClassService()`